src/incomingDelivery/controller.js

```diff
@@ -10,6 +13,17 @@ router.get("/", getAll);
 router.put("/", createOne);
 router.get("/queue", getQueue);
 router.post("/receive", setReceived);
+// Make sure the editMadeBy is valid
+router.post("/undoReceive", async (req, res, next) => {
```

Comment:
`editMadeBy = req.user._id` with how passport is configured in our project.
This block is unnecessary.